src/_layouts/default.html

```diff
@@ -55,5 +31,8 @@
       {% include page-content.html %}
     </main>
     {% include contribute.html %}
+    <script src="/assets/js/plausible.js">
+      plausible('pageview', {u: prepareUrl([])})
+    </script>
```

Comment:
Does this script need to execute at the end of the file rather than in the `head`?